internal/service/service.go

```diff
@@ -38,6 +35,8 @@ func DepGraphToSBOM(
 	}
 	req.Header.Add("Content-Type", MimeTypeJSON)
 
+	logger.Printf("Converting depgraph remotely (url: %s)", req.URL.String())
```

Comment:
@mcombuechen does this get displayed to the user? If so, I think it should probably be a debug message, what do you think?

Reply:
Yes it gets displayed to the user once they set the `--debug` flag. This applies to all messages printed via the `logger`.

Reply:
@mcombuechen ok that pretty much clears up my comments then, approvd